src/util/colorScale.ts

```diff
@@ -309,7 +309,7 @@ function createTemporalScale(
     vals = vals.sort();
     domain = [rootDate];
     const n = 10;
-    const spaceBetween = parseInt(vals.length / (n - 1), 10);
+    const spaceBetween = vals.length / (n - 1);
```

Comment:
There's a bug with this change which results in temporal colour scales failing (everything's black) when `vals.length` is not an integer multiple of `(n-1)` - `spaceBetween` must be a integer so that we can use it as an array index below (`vals[spaceBetween*i`).

Reply:
Didn't know about `Math.trunc` but it looks perfect. I've used `parseInt(String(x), 10)` in the past to appease linters but in reality `parseInt` will cast to a string anyway.